calico_cni_k8s_test.go

```diff
@@ -885,6 +885,55 @@ var _ = Describe("CalicoCni", func() {
 					_, err = DeleteContainer(netconf, netnspath, "")
 					Expect(err).ShouldNot(HaveOccurred())
 				})
+
+				Context("when IPAM returns an empty result", func() {
+					It("should try to release any IP addresses", func() {
+						netConf := fmt.Sprintf(`
+				{
+			      "cniVersion": "%s",
+				  "name": "net2",
+				  "type": "calico",
+				  "etcd_endpoints": "http://%s:2379",
+			 	  "ipam": {
+			    	 "type": "echo-ipam",
+				 "return": "{}"
```

Comment:
I don't really get this line, is it left over from a previous design of your echo plugin?